src/EDLog/EDLog.ts

```diff
@@ -358,10 +358,13 @@ export class EDLog extends EventEmitter {
 
     /**
      * Launches the log reader.
-     * @param If true the method will return an array of event logs, otherwise empty.
+     * 
+     * @param backlog Optional configuration settings.
+     * @returns {EDEvent[]} The processed backlog events.
```

Comment:
`@returns {Type}` is entirely redundant here, we have TS for a reason, I also believe the VSCode generally ignores `@returns` docs.